src/encoded/static/components/facets/facet_groups.js

```diff
@@ -0,0 +1,212 @@
+    return (
+        <button
+            type="button"
+            className={`facet-group-title${isExpanded ? ' facet-group-title--open' : ''}`}
+            aria-expanded={isExpanded}
+            aria-controls={generateFacetGroupIdentifier(group)}
+            onClick={handleClick}
+        >
+            <div className="facet-group-title__title-name">
+                <div className="facet-group-title__title">
+                    {group.title}
+                    {hasSelections && (
+                        <div className="facet-group-title__selection-indicator">
+                            {svgIcon('circle')}
+                        </div>
+                    )}
+                </div>
+                {isNameDisplayed && (
+                    <div className="facet-group-title__name">
+                        {profileName || group.name}
+                    </div>
+                )}
+            </div>
+            <div className="facet-group-title__expand-indicator">
+                {svgIcon(isExpanded ? 'chevronUp' : 'chevronDown')}
+            </div>
+        </button>
+    );
+};
+
+FacetGroupTitle.propTypes = {
+    /** Section definition from schema */
+    group: PropTypes.object.isRequired,
+    /** True if section appears expanded */
+    isExpanded: PropTypes.bool.isRequired,
+    /** True if at least one facet within this group has selections */
+    hasSelections: PropTypes.bool.isRequired,
+    /** True to display the schema name below the title */
+    isNameDisplayed: PropTypes.bool.isRequired,
+    /** Called to handle clicks in the section title */
+    expanderHandler: PropTypes.func.isRequired,
+};
+
+FacetGroupTitle.contextTypes = {
+    profilesTitles: PropTypes.object,
+};
+
+
+/**
+ * Renders one facet group when searched objects have a schema with a `facet_groups` property.
+ */
+export const FacetGroup = ({
+    group,
+    filters,
+    isExpanded,
+    isNameDisplayed,
+    expanderHandler,
+    children,
+}) => {
+    // Determine whether filters include any fields within this group.
+    const hasSelections = filters.some((filter) => {
+        // Ignore trailing exclamation point on negative filters.
+        const filterField = filter.field.slice(-1) === '!' ? filter.field.slice(0, -1) : filter.field;
+        return group.facet_fields.includes(filterField);
+    });
+
+    return (
+        <div className="facet-group">
+            <FacetGroupTitle group={group} hasSelections={hasSelections} isExpanded={isExpanded} isNameDisplayed={isNameDisplayed} expanderHandler={expanderHandler} />
+            <AnimatePresence>
+                {children &&
+                    <motion.div
+                        id={generateFacetGroupIdentifier(group)}
+                        className="facet-group__content"
+                        initial="collapsed"
+                        animate="open"
+                        exit="collapsed"
+                        transition={{ duration: 0.2, ease: 'easeInOut' }}
```

Comment:
This could import from defaults.js. Just in case we ever want to tweak the transition timing.